packages/charting/package.json

```
@@ -53,6 +53,7 @@
     "sinon": "^4.1.3"
   },
   "dependencies": {
+    "@fluentui/react-focus": "7.7.0",
```

Looks like you're getting a failure because this doesn't match what's in production and it's hardcoded. I believe that there is some work done to ensure that all packages within the fluent repo stay in sync. @ecraig12345 might know more.

```suggestion
    "@fluentui/react-focus": "^7.7.0",
```

---

It just needs to match the version elsewhere in the repo. react-focus was published again since this change was made, so you need to do `^7.7.1` now.

---

I have updated to ^7.7.1